### PR TITLE
[management] Updated READMEs for operational tools

### PR DESCRIPTION
**Pull request by gregnazario**
Added a quick run at some READMEs, I think we can add more depth to them as we go along.

### CODE REVIEW

File: config/management/README.md [hunk 1]
```diff
@@ -1,190 +1,14 @@
-# Libra Config Manager
+# Libra Management tools
 
-The Libra Config Manager provides a tool for end-to-end management of the Libra
-blockchain from genesis to maintenance. The functionality of the tool is
-dictated by the organization of nodes within the system:
+The `libra-management` crate provides a framework for building CLI tools for various
+purposes.  The purpose of breaking these into multiple tools is to simplify the user
+experience, and prevent confusion between the different situations.
```
Comment (JoshLind) [inline config/management/README.md:5]: nit: different situations -> different use cases?

File: config/management/README.md [hunk 2]
```diff
@@ -1,190 +1,14 @@
...
-```
-cargo run -p libra-management -- \
-    validator-config \
-    --owner-name OWNER_NAME \
-    --validator-address '/dns/DNS/tcp/PORT' \
-    --fullnode-address '/dns/DNS/tcp/PORT' \
-    --local 'backend=vault;server=URL;token=PATH_TO_VAULT_TOKEN' \
-    --remote 'backend=github;repository_owner=REPOSITORY_OWNER;repository=REPOSITORY;token=PATH_TO_GITHUB_TOKEN;namespace=NAME'
+libra-management
+|-> libra-genesis-tool = A tool for managing genesis of a Libra blockchain.
```
Comment (JoshLind) [inline config/management/README.md:10]: nit: maybe "A tool for performing the genesis ceremony for the Libra blockchain"

File: config/management/README.md [hunk 3]
```diff
@@ -1,190 +1,14 @@
...
+|-> libra-operational-tool = A tool for managing an already running Libra blockchain.
```
Comment (JoshLind) [inline config/management/README.md:11]: nit: maybe "A tool for performing management operations on the Libra blockchain"

File: config/management/genesis/README.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
+# Libra Config Manager
+
+The Libra Config Manager provides a tool for end-to-end management of the Libra
```
Comment (JoshLind) [inline config/management/genesis/README.md:3]: nit: Maybe update this description to be more specific about genesis and remove "maintenance"?

File: config/management/operational/README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+The Libra Config Manager provides a tool for operators to perform maintenance functions
+on the Libra blockchain post-genesis.  The functionality of the tool is
+dictated by the organization of nodes within the system:
+
+* An association account that maintains the set of validator owners and validator
+  operators.
+* Validator owners (OW) that have accounts on the blockchain. These accounts contain
+  a validator configuration and specify a validator operator.
+* Validator operators (OP) that have accounts on the blockchain. These
+  accounts have the ability to manipulate validator configuration.
```
Comment (JoshLind) [inline config/management/operational/README.md:12]: nit: "manipulate" -> "update"

File: config/management/operational/README.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+solution. Those leveraging Libra Secure Storage can directly use this tool,
+those that do not will need to provide their own tooling.
+
+## The Tools
+
+While this is compiled as a single binary it provides several different facilities:
+
+* A means for bootstrapping an identity mapped to a local configuration file.
+  The identities are used to interact with local and remote secure storages.
+* Retrieving and submitting validator operator, validator operator, and validator
```
Comment (JoshLind) [inline config/management/operational/README.md:26]: typo: "submitting validator operator, validator operator, and .."

File: config/management/genesis/README.md (hunk 4 above)
Comment (gregnazario) [reply config/management/genesis/README.md:3]: Ah, I straight copied it from the previous README

File: config/management/operational/README.md (hunk 6 above)
Comment (gregnazario) [reply config/management/operational/README.md:26]: lol, also looks like this was straight copied from the previous README

**Pull request closed without merging**